config/locales/zh-TW.yml

```diff
@@ -0,0 +1,163 @@
+zh-TW:
+  title: "RubyGems.org"
+  subtitle: "Ruby 社群 Gem 套件管理平台"
+  help_url: "http://help.rubygems.org"
+  please_sign_up: "無法存取，請先在 https://rubygems.org 上註冊帳號"
+  feed_subscribed: "RubyGems.org | 訂閱 Gems"
+  feed_latest: "RubyGems.org | 最新 Gems"
+  time_ago: "%{duration} 前"
+  form_disable_with: "請稍候..."
+  update: "更新"
+  edit: "編輯"
+  sign_in: "登入"
+  sign_up: "註冊"
+  footer_about: "RubyGems.org 是 Ruby 社群的 Gem 套件管理服務，讓你能立即地發佈及安裝你的 Gem 套件，並且利用 API 查詢及操作可用 Gem 的詳細資訊。<br />分享你的貢獻，讓 RubyGems.org 變得更加完善。"
+
+  dashboards:
+    show:
+      latest: "最近更新"
+      latest_title: "最新的 RSS Feed"
+      no_subscriptions: "你還沒有訂閱過 Gem，前往 %{gem_link} 來訂閱！"
+      gem_link_text: "Gem 頁面"
+      mine: "我的 Gems"
+      no_owned: "你還沒有發佈過 Gem，請參考%{creating_link}教學，或從 RubyForge 的%{migrating_link}教學。"
+      creating_link_text: 'Gem 建立'
+      migrating_link_text: 'Gem 轉移'
+      my_subscriptions: '我的訂閱'
+
+  home:
+    index:
+      downloads_counting: '總下載次數'
+      find_blurb: "搜尋、安裝以及發佈 RubyGems"
+      learn:
+        install_rubygems: "安裝 RubyGems"
+
+  layouts:
+    application:
+      header:
+        dashboard: "控制台"
+        sign_in: "登入"
+        sign_up: "註冊"
+        sign_out: "登出"
+      footer:
+        about: "關於"
+        blog: "部落格"
+        guides: "教學文件"
+        contribute: "貢獻"
+        help: "問題 / 回報"
+        status: "Status"
+        uptime: "Uptime"
+        source_code: "原始碼"
+        data: "資料"
+        discussion_forum: "討論群組"
+        statistics: "統計"
+
+  pages:
+    about:
+      title: 關於
+      purpose:
+        header: "歡迎到 %{site}，這裡是 Ruby 社群 Gem 套件管理服務。此專案的建立有以下三個目的:"
+        better_api: "提供更好的 Gem API"
+        transparent_pages: "建立更透明、更易於瀏覽的 Gem 專案頁面"
+        enable_community: "讓社群能透過眾人的力量來改善本站"
+      founding: "這個專案是由 %{founder} 於 2009 年 4 月建立，發展過程中有超過 %{contributors} 貢獻者加入以及 %{downloads}。 而自從 RubyGems 1.3.6 發布之後，本站的名稱由 Gemcutter 改名為 %{title}，表示本站在 Ruby 社群中的核心角色。"
```

Comment:
I would translate to this:

```
本專案由 %{founder} 於 2009 年 4 月創立，發展過程中有超過 %{contributors} 貢獻者以及 %{downloads}。自 RubyGems 1.3.6 發佈以來，本站名稱由 Gemcutter 更名為 %{title}，本站自此之後成為 Ruby 社群的核心網站。
```

Could you see if any could be incorporated into yours from my above translation?